packages/xstate-wallet/src/store/state-utils.ts

```diff
@@ -41,6 +43,27 @@ function toNitroState(state: State): NitroState {
   };
 }
 
+export function fromNitroState(state: NitroState): State {
+  const {appData, isFinal, outcome, challengeDuration, appDefinition, channel, turnNum} = state;
+
+  return {
+    appDefinition,
+    isFinal,
+    appData,
+    outcome: fromNitroOutcome(outcome),
+    turnNum: bigNumberify(turnNum),
+    challengeDuration: bigNumberify(challengeDuration),
+    channelNonce: bigNumberify(channel.channelNonce),
+    chainId: channel.chainId,
+    participants: channel.participants.map(x => ({
+      signingAddress: x,
+      // FIXME: Get real values
+      participantId: x,
+      destination: x as Destination
```

Comment:
Not 100% sure, but I think that maybe `Destination`s need to be padded to `Bytes32`.